packages/cli/src/commands/migrate.ts

```diff
@@ -44,11 +45,18 @@ type ParsedModuleResult = {
   coreDepList: Array<string>;
 };
 
+type FormatterFunction = (report: Report) => string;
+
+type FormatterMap = {
+  [format: string]: FormatterFunction;
+};
+
 migrateCommand
   .description('Migrate Javascript project to Typescript')
   .requiredOption('-b, --basePath <project base path>', 'Base dir path of your project.')
   .requiredOption('-e, --entrypoint <entrypoint>', 'entrypoint js file for your project')
-  .option('-r, --report', 'create report for migration result')
+  .option('-r, --report <reportFormat>', 'Report formats separated by comma, e.g. -r json,sarif,md')
```

Comment:
You can use [variadic option](https://www.npmjs.com/package/commander#variadic-option) or see [commaSeparatedList](https://www.npmjs.com/package/commander#custom-option-processing)

Reply:
agreed. I wonder if we can do the same with the flag directions eg. `Report formats separated by comma, e.g. -r <reportTypes>`

Reply:
I went with [Custom option processing](https://www.npmjs.com/package/commander#custom-option-processing) to handle the comma-separated args, but open to change to `variadic` if you guys prefer it.